### PR TITLE
Fix Geo Target Report bug + Add missing tests

### PR DESCRIPTION
**Pull request by tabrethauer**

### CODE REVIEW

File: common/sheet_helpers.ts [hunk 1]
```diff
@@ -114,7 +114,7 @@ export class SettingMap<P extends { [Property in keyof P]: P[keyof P] }>
     return this.keys.reduce(
       (prev, key) => {
         prev[key] =
-          (!(key in campaignValue) || campaignValue[key] === ''
+          (!(key in campaignValue) || !campaignValue[key]
```
Comment (smcjones) [inline common/sheet_helpers.ts:117]:
```suggestion
          (!(key in campaignValue && campaignValue[key])
```

What do you think @tabrethauer ? I know I wrote this line.

Comment (smcjones) [reply common/sheet_helpers.ts:117]: sticking with this as I think it's DNF

**Review by anaesqueda — APPROVED** [review]
LGTM, can we also deploy all the changes to a sheet template (the main template) so they can be tested end to end? Unit tests help us to catch bugs in specific modules/ logic and end to end tests will help to catch other issues in the integration. 
Thanks!

**Comment by smcjones** [comment]
> LGTM, can we also deploy all the changes to a sheet template (the main template) so they can be tested end to end? Unit tests help us to catch bugs in specific modules/ logic and end to end tests will help to catch other issues in the integration. Thanks!

Yeah, we should probably make this a required step before merging.

**Pull request merged**